src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -1462,24 +1462,6 @@ void primitive_inst::configure_shape_of_dependencies() {
     }
 }
 
-void primitive_inst::rebuild_deps(std::unordered_map<primitive_id, primitive_inst*> const& primitives) {
-    _deps.resize(_dep_ids.size());
-    for (size_t i = 0; i < _dep_ids.size(); i++) {
-        OPENVINO_ASSERT((primitives.count(_dep_ids[i].first) > 0),
-                        _dep_ids[i].first, "is not found in primitives while rebuilding _deps");
-        _deps[i] = {primitives.at(_dep_ids[i].first), _dep_ids[i].second};
-    }
-}
-
-void primitive_inst::rebuild_exec_deps(std::unordered_map<primitive_id, primitive_inst*> const& primitives) {
-    _exec_deps.resize(_exec_dep_ids.size());
-    for (size_t i = 0; i < _exec_dep_ids.size(); i++) {
-        OPENVINO_ASSERT((primitives.count(_exec_dep_ids[i]) > 0),
-                        _exec_dep_ids[i], "is not found in primitives while rebuilding _exec_deps");
-        _exec_deps[i] = primitives.at(_exec_dep_ids[i]);
-    }
-}
```

Comment:
Could you explain how this change resolves the original issue?

Reply:
This change is not relevant with the original issue. It's just a code cleanup. Thank you.